package.json

```diff
@@ -9,6 +9,8 @@
   "author": "DuckDuckGo",
   "license": "MIT",
   "devDependencies": {
-    "grunt": "^0.4.5"
+    "grunt": "^1.0.1",
```

Comment:
I take it this version bump is necessary for grunt webfont. Any idea if v1 has any breaking changes?